rules/do-you-know-how-to-manage-qantas-bookings/rule.md

```diff
@@ -1,34 +1,43 @@
 ---
 type: rule
-title: Do you know how to better manage Qantas bookings?
-uri: do-you-know-how-to-manage-qantas-bookings
+title: Do you know how to retrieve missing invoices for flights?
+uri: do-you-know-how-to-retrieve-missing-invoices-for-flights
```

Comment:
Hey @JimmyChenSSW 

Can you follow all steps to rename a rule: https://github.com/SSWConsulting/SSW.Rules.Content/wiki/How-to-Edit-Rules#renaming-rules

Tip: 1. You need a redirect + 2. fix folder name + 3. update the uri in the category

Reply:
PS. I just saw you actioned item 3 on #5062

Reply:
also keep in mind this rule URL is too long, I made a recommendation in the other PR.